tmt/steps/prepare/install.py

```diff
@@ -229,7 +229,11 @@ def go(self, guest):
             packages = ' '.join(
                 [tmt.utils.quote(package) for package in repo_packages])
             check = f'rpm -q --whatprovides {packages}'
-            # Check and install (extra check for yum to workaround BZ#1920176)
+            # Extra ignore/check for yum to workaround BZ#1920176
+            if 'yum' in command:
+                yum = " || true" if skip else f" && {check}"
```

Comment:
Name of this variable is a bit confusing. Can't think of a good name now :/ (tail, finish, after...)

Reply:
Same with me... `hack`? `recheck`? `extra_check`? `double_check`? `ensure`? `inspect`? `examine`?

Reply:
what about `yum_fix`?

Reply:
Sounds good, updated.

Reply:
`yum_check`?

Reply:
Ok, this seems to be the best candidate. Changed.